src/xcode/ENA/ENA/Source/Scenes/ENSetting/Cells/ActionDetailTableViewCell.swift

```diff
@@ -90,9 +90,17 @@ class ActionDetailTableViewCell: UITableViewCell {
 		if let state = self.state, state == .unknown {
 			delegate?.performAction(action: .askConsent)
 		} else {
-			if let settingsUrl = URL(string: UIApplication.openSettingsURLString),
-				UIApplication.shared.canOpenURL(settingsUrl) {
-				UIApplication.shared.open(settingsUrl, completionHandler: nil)
+			let url: URL?
+			if state == .bluetoothOff {
+				// this will open the os settings
+				url = URL(string: "App-Prefs:root=General")
```

Comment:
I think this could be rejected.

Reply:
If that is the case then we don't have any possibility to open the OS settings and the corresponding ticket should be marked as invalid.

Reply:
We had a similar story before, where it was requested to open the general settings instead of the app settings. We rejected the story because of the risk being rejected.

Reply:
Could we just open the apps settings instead? See https://github.com/corona-warn-app/cwa-app-ios/pull/2237#issuecomment-801742523